src/chunk.c

```diff
@@ -242,21 +308,31 @@ static void Uncompressed_GenericSerialize(Chunk_t *chunk,
     saveUnsigned(ctx, uncompchunk->base_timestamp);
     saveUnsigned(ctx, uncompchunk->num_samples);
     saveUnsigned(ctx, uncompchunk->size);
-
-    saveString(ctx, (char *)uncompchunk->samples, uncompchunk->size);
+    Sample *old_samples_array = (Sample *)malloc(uncompchunk->size * sizeof(Sample));
```

Comment:
@filipecosta90 don't you prefer to create a new RDB version that will have a different format?

Reply:
@danni-m agree. Will work on it ( same like we want for compressed )